subprojects/gradle/build-checks/build.gradle.kts

```diff
@@ -28,7 +28,7 @@ dependencies {
 gradlePlugin {
     plugins {
         create("buildChecks") {
-            id = "com.avito.android.buildchecks"
+            id = "com.avito.android.build-checks"
```

Comment:
It will break backward compatibility for clients.

Also, need to change it in docs and tests

Reply:
changed